configurer/linux.go

```diff
@@ -136,6 +136,12 @@ func (l Linux) ReplaceK0sTokenPath(h os.Host, spath string) error {
 	return h.Exec(fmt.Sprintf("sed -i 's^REPLACEME^%s^g' %s", l.PathFuncs.K0sJoinTokenPath(), spath))
 }
 
+// FileExists returns true if a file exiyts
+func (l Linux) FileExists(h os.Host, path string) bool {
+	err := h.Execf(fmt.Sprintf("stat %s", path), exec.Sudo(h))
+	return err == nil
+}
+
```

Comment:
This already exists in rig: https://github.com/k0sproject/rig/blob/main/os/linux.go#L230-L233

But calling those is not easy. I think you need something like:

```go
import(
  rigos "github.com/k0sproject/rig/os"
)

...

if rigos.Linux{}.FileExist(h, path) {
   ...
}
```

Would be nice to come up with a better way to compose rig but can't think of one.

Reply:
I was wondering if rig had something like it but couldn't see it on the `os.Host` interface.